python/CMakeLists.txt

```diff
@@ -36,21 +35,42 @@ pybind11_add_module(
   element_array_view.cpp
   shape.cpp
 )
+pybind11_add_module(_scipp_neutron SHARED SYSTEM neutron.cpp scipp_neutron.cpp)
```

Comment:
You intend to change this next? Maybe better discussed on a call.

Reply:
Correct. Will first copy the neutron bits from `scipp` into `scippneutron`. Once that builds we will remove it here.